proposals/PROPOSALS.md

```diff
@@ -0,0 +1,156 @@
+
+The maximum number of iterations is five. At the fifth iteration, the HF Technical Track can only vote to Accept, Postpone, or Reject.
+
+The HF Technical Track decides the duration of the next iteration depending upon the feedback and complexity of the HIP. Consequently, authors have more time to prepare all the changes. If they finish their revision before the scheduled iteration, an HF Technical Track member will reschedule it for the next available meeting.
+
+During every iteration, the HF Technical Track reviews the changes (updated design document, progress with the implementation, etc) to the proposal. Based on the feedback, the HIP is either:
+
+1. **Accepted**, in which case the HF Technical Track has accepted the proposal, and it will be merged by the HF Technical Track members.
+2. **Rejected**, in which case the HIP is closed and no longer evaluated in the future.
+3. **Postponed**, in which case the HF Technical Track sets aside the HIP under some conditions. When those conditions are met, the HIP can be resubmitted.
+4. **Under revision**, in which case the author needs to continue the formal evaluation and address all the HF Technical Track feedback. Thus, the follow-up discussion is scheduled for the next iteration. 
+5. **Dormant**,  in which case no changes have been made to a HIP in two iterations, it’s marked as dormant and both the PR and issue are closed. Dormant HIPs can be reopened by any person, be it the same or different authors, at which point it will start from the formal evaluation phase.
+
+#### Involving Stakeholders
+
+At this point where a propsal is being reviewed, if there are relevant stakeholders from industry or in the community (e.g. library authors and maintainers who are affected), they should be solicited for comment and brought into the discussion. Remember, proposals may sound good, but it's best to iron out as many details as possible prior to their acceptance. This means that any hesitations stakeholders have with the project should be factored into the proposal's details. 
+
+#### Updating Proposal Statuses
+
+If the author of a particular proposal wants to update the status of their proposal, they should either comment on the issue and tag an HF Technical Track member with a note regarding what status they would like to update the proposal to. It is good to be redundant and raise the question in our other fora as well: Slack and Discourse. However, Github should be the first choice. 
+
+### Formal presentation 
+
+After a proposal is merged, the author will present KPI's and deliverables that define the proposal to the HF Technical track. During the next available Technical Track meeting after acceptance, the author or designated lead (as defined by the proposal) should give a short 5-10min presentation consisting of the following: 
```

Comment:
What is a KPI? I really don't know.

This step seems surprising to me. During the whole deliberation process, it seems that the author is not generally invited to present their case. And then after the proposal is accepted, they can? I'm not sure I see the logic here.

Reply:
I too am a bit surprised.  I'm inclined to suggest that this document ends with acceptance. Then there is some separate process by which the HF board, led by its ED and CTO have to prioritise resources, choose initiatives, etc.   That may or may not involve project presentations, but it'll involve a lot else.  But it's beyond the scope of this document. 

This links with my remarks below about the distinction between acceptance and execution.

Reply:
> What is a KPI

Key Performance Indicators - it's a term used in project management in Agile settings. I'll just spell it out. 

> Suprising

The logic here is that the accepted proposal represents the finalized proposal state, while the presentation gives the authors a chance to present the finalized proposal to the track as a kind of "first standup". The HFTT gets to meet the representative who will be giving future updates, as well as give the broader HFTT (who may all not have seen the proposal) the chance to see the finished product. Of course there are questions here: is this necessary? Perhaps not. I may have gone beyond the scope here.

Reply:
Yes, I tend to think that this is beyond the process. That is, once we agree to pursue a project, the proposals process is complete. It's quite reasonable for the HFTT to both oversee the proposal process and the implementation process, but the meeting described here sounds like part of the implementation process. In my opinion, there's not as much of a need to codify implementation processes, as there will be big differences between e.g. volunteer work and work that the HF pays for.

However, there is still an open question: is there an opportunity for proposal authors to make their case live to the HFTT? Maybe this step is optional, but if so, who decides whether authors get to present? Does the HFTT invite them based on their interest? Does the author decide they wish to attend in person? I tend to think it's fairest if we give proposers the opportunity to make their case to us -- all the more so because I suspect that HFTT members will sometimes also be authors, and naturally HFTT members will advocate for their proposals in meetings.